### PR TITLE
Update photon image

### PR DESCRIPTION
**Pull request by gerrit-photon**
Update photon image

### CODE REVIEW

File: library/photon [hunk 1]
```diff
@@ -1,14 +1,74 @@
...
 GitRepo: https://github.com/vmware/photon-docker-image.git
 Directory: docker
 
-Tags: 2.0, 2.0-20180704, latest
+Tags: 2.0-20180704, 2.0, latest
 GitFetch: refs/heads/2.0-20180704
 GitCommit: e6b78e76f36da84a67c5b8e411a50514a1b62357
 
-Tags: 1.0, 1.0-20180703
+Tags: 2.0-20180703
```
Comment (yosifkit) [inline library/photon]: Any particular reason to add all the old versions listed here?  Removing it here will remove it from the "Supported" section on the Hub readme, but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Comment (gerrit-photon) [reply library/photon]: oh, sorry. let me check. Looks like script make it mess a little.

**Comment by gerrit-photon** [comment]
photon docker file has been updated. Could you please help to review it?

Thanks,
Michelle

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 672203b..4bf334e 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,2 +1,2 @@
-photon:1.0-20180703 @ amd64
+photon:1.0-20180709 @ amd64
 photon:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index 8e2b9de..2a28730 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,5 +1,5 @@
 photon:1.0
-photon:1.0-20180703
+photon:1.0-20180709
 photon:2.0
-photon:2.0-20180704
+photon:2.0-20180710
 photon:latest
diff --git a/photon_1.0-20180703/Dockerfile b/photon_1.0-20180709/Dockerfile
similarity index 59%
rename from photon_1.0-20180703/Dockerfile
rename to photon_1.0-20180709/Dockerfile
index 178a2cc..572019d 100644
--- a/photon_1.0-20180703/Dockerfile
+++ b/photon_1.0-20180709/Dockerfile
@@ -1,8 +1,8 @@
 FROM scratch
-ADD photon-rootfs-1.0-d694d7b.tar.bz2 /
+ADD photon-rootfs-1.0-409b661.tar.bz2 /
 
 LABEL name="Photon OS 1.0 Base Image" \
     vendor="VMware" \
-    build-date="20180703"
+    build-date="20180709"
 
 CMD ["/bin/bash"]
diff --git a/photon_1.0-20180703/photon-rootfs-1.0-d694d7b.tar.bz2 b/photon_1.0-20180709/photon-rootfs-1.0-409b661.tar.bz2
similarity index 78%
rename from photon_1.0-20180703/photon-rootfs-1.0-d694d7b.tar.bz2
rename to photon_1.0-20180709/photon-rootfs-1.0-409b661.tar.bz2
index 371f5bc..3da5e92 100644
Binary files a/photon_1.0-20180703/photon-rootfs-1.0-d694d7b.tar.bz2 and b/photon_1.0-20180709/photon-rootfs-1.0-409b661.tar.bz2 differ
diff --git a/photon_1.0-20180703/photon-rootfs-1.0-d694d7b.tar.bz2  'tar -t' b/photon_1.0-20180709/photon-rootfs-1.0-409b661.tar.bz2  'tar -t'
similarity index 100%
rename from photon_1.0-20180703/photon-rootfs-1.0-d694d7b.tar.bz2  'tar -t'
rename to photon_1.0-20180709/photon-rootfs-1.0-409b661.tar.bz2  'tar -t'
diff --git a/photon_latest/Dockerfile b/photon_latest/Dockerfile
index 34501d2..7c44e3e 100644
--- a/photon_latest/Dockerfile
+++ b/photon_latest/Dockerfile
@@ -1,8 +1,8 @@
 FROM scratch
-ADD photon-rootfs-2.0-1de4adc.tar.bz2 /
+ADD photon-rootfs-2.0-678bc40.tar.bz2 /
 
 LABEL name="Photon OS 2.0 Base Image" \
     vendor="VMware" \
-    build-date="20180704"
+    build-date="20180710"
 
 CMD ["/bin/bash"]
diff --git a/photon_latest/photon-rootfs-2.0-1de4adc.tar.bz2 b/photon_latest/photon-rootfs-2.0-678bc40.tar.bz2
similarity index 60%
rename from photon_latest/photon-rootfs-2.0-1de4adc.tar.bz2
rename to photon_latest/photon-rootfs-2.0-678bc40.tar.bz2
index b8a924d..841143c 100644
Binary files a/photon_latest/photon-rootfs-2.0-1de4adc.tar.bz2 and b/photon_latest/photon-rootfs-2.0-678bc40.tar.bz2 differ
diff --git a/photon_latest/photon-rootfs-2.0-1de4adc.tar.bz2  'tar -t' b/photon_latest/photon-rootfs-2.0-678bc40.tar.bz2  'tar -t'
similarity index 100%
rename from photon_latest/photon-rootfs-2.0-1de4adc.tar.bz2  'tar -t'
rename to photon_latest/photon-rootfs-2.0-678bc40.tar.bz2  'tar -t'
```

</details>

**Comment by yosifkit** [comment]
Build test of #4558; 0c124fa1; `amd64` (`photon`):

```console

$ bashbrew build photon:2.0
Building bashbrew/cache:b9cd5975c75200f486191ee470760811c7bcfb940701ac42c6206ad265f51d23 (photon:2.0)
Tagging photon:2.0
Tagging photon:2.0-20180710
Tagging photon:latest

$ test/run.sh photon:2.0
testing photon:2.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...cut: /etc/shadow: No such file or directory
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'systemd-timesync': 'x'
warning: garbage password detected for 'systemd-resolve': 'x'
warning: garbage password detected for 'systemd-network': 'x'
warning: garbage password detected for 'messagebus': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'systemd-journal-upload': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'systemd-bus-proxy': 'x'
warning: garbage password detected for 'systemd-journal-remote': 'x'
warning: garbage password detected for 'systemd-journal-gateway': 'x'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build photon:1.0
Using bashbrew/cache:b6215582eba90c7d8f471f40ab77f24b30671f34353be9075b57e481cdb8480c (photon:1.0)
Tagging photon:1.0
Tagging photon:1.0-20180709

$ test/run.sh photon:1.0
testing photon:1.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...cut: /etc/shadow: No such file or directory
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'systemd-timesync': 'x'
warning: garbage password detected for 'systemd-resolve': 'x'
warning: garbage password detected for 'systemd-network': 'x'
warning: garbage password detected for 'messagebus': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'systemd-journal-upload': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'systemd-bus-proxy': 'x'
warning: garbage password detected for 'systemd-journal-remote': 'x'
warning: garbage password detected for 'systemd-journal-gateway': 'x'
passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**